addons/test_mail/tests/test_mail_activity.py

```diff
@@ -759,3 +761,253 @@ def test_week_grouping(self):
         self.assertEqual(groups[0][groupby], pg_groups["overdue"])
         self.assertEqual(groups[1][groupby], pg_groups["today"])
         self.assertEqual(groups[2][groupby], pg_groups["planned"])
+
+
+class TestActivitySchedule(TestActivityScheduleCommon):
+    """ Test plan and activity schedule
+
+     We test:
+     - activity scheduling on a single record and in batch
+     - plan scheduling on a single record and in batch
+     - plan creation and consistency
+     """
+
+    def check_action_feedback_schedule_plan(self, action, records):
```

Comment:
Not sure it really deserves to be tested (makes some noise and added code just for a returned action)